src/Sulu/Bundle/AdminBundle/Resources/js/containers/Datagrid/AdapterSwitchItem.js

```diff
@@ -0,0 +1,34 @@
+// @flow
+import React from 'react';
+
+type Props = {
+    adapter: string,
+    active: boolean,
+    onClick: (adapter: string) => void,
+};
+
+export default class AdapterSwitchItem extends React.PureComponent<Props> {
+    handleClick = () => {
+        const {
+            adapter,
+            onClick,
+        } = this.props;
+
+        onClick(adapter);
+    };
+
+    render() {
+        const {
+            active,
+            adapter,
+        } = this.props;
+
+        return (
+            <li key={adapter} onClick={this.handleClick}>
```

Comment:
`key` doesn't make any sense here. It is only needed in loops, not in the implementation of the element being created in a loop.